website/docs/d/schema_site_service_graph.html.markdown

```diff
@@ -34,3 +34,8 @@ data "mso_schema_site_service_graph" "example" {
 
 * `service_node` - (Read-Only) List of maps to provide Site level Node association.
     * `device_dn` - (Read-Only) Dn of device associated with the service node of the Service Graph.
+    * `provider_connector_type` - (Read-Only) Connector type of the provider type of the service node. This parameter is only applicable for cloud sites.
+    * `consumer_connector_type` - (Read-Only) Connector type of the consumer type of the service node. This parameter is only applicable for cloud sites.
+    * `firewall_provider_connector_type` - (Read-Only) Connector type of the provider type of the third-party firewall service node.
```

Comment:
Do we need the parameter applicable line in data-source? If so, "This parameter is only applicable for third-party firewall service nodes, when the template is attached to cloud sites." should be added since this is also done in resource docs.

Reply:
Yeah, I think might be good to add it to the data source docs to keep it consistent with the resource docs.  :)